shared-module/vectorio/Circle.c

```diff
@@ -23,7 +23,7 @@ void common_hal_vectorio_circle_set_on_dirty(vectorio_circle_t *self, vectorio_e
 
 uint32_t common_hal_vectorio_circle_get_pixel(void *obj, int16_t x, int16_t y) {
     vectorio_circle_t *self = obj;
-    int16_t radius = abs(self->radius);
+    int16_t radius = self->radius;
```

Comment:
This is the only thing I'd currently advocate for changing in here. This change looks great to me, thanks 😄

You _could_ use the value as-is instead of the typecast it's doing to a signed integer but no big deal either way imo:
```suggestion
    uint16_t radius = self->radius;
```